wagtail/admin/templates/wagtailadmin/home/workflow_objects_to_moderate.html

```diff
@@ -44,31 +44,25 @@
                                 </div>
                                 {% if actions %}
                                     <ul class="actions">
-                                        {% for action_name, action_label, modal in actions %}
-                                            <li>
-                                                <button class="button button-small button-secondary" data-workflow-action-url="{% url state.workflow_action_url_name obj.pk|admin_urlquote action_name task_state.id %}" {% if modal %}data-launch-modal{% endif %}>{{ action_label }}</button>
-                                            </li>
-                                        {% endfor %}
-                                        {% if page_perms.can_edit or not is_page and edit_url %}
-                                            <li>
-                                                <a href="{{ edit_url }}" class="button button-small button-secondary">{% trans 'Edit' %}</a>
-                                            </li>
-                                        {% endif %}
-                                        {% if obj.is_previewable %}
-                                            <li>
-                                                <a href="{% url state.workflow_preview_url_name obj.pk|admin_urlquote task_state.task.id %}" class="button button-small button-secondary" target="_blank" rel="noreferrer">{% trans 'Preview' %}</a>
-                                            </li>
-                                        {% endif %}
-                                        {% if state.live_revision %}
-                                            <li>
-                                                <a href="{% url state.revisions_compare_url_name obj.pk|admin_urlquote 'live' revision.id %}" class="button button-small button-secondary">{% trans "Compare with live version" %}</a>
-                                            </li>
-                                        {% endif %}
-                                        {% if state.previous_revision %}
-                                            <li>
-                                                <a href="{% url state.revisions_compare_url_name obj.pk|admin_urlquote state.previous_revision.id revision.id %}" class="button button-small button-secondary">{% trans "Compare with previous version" %}</a>
-                                            </li>
-                                        {% endif %}
+                                        <li>
+                                            {% dropdown toggle_icon="dots-horizontal" toggle_aria_label=_("Actions") hide_on_click=True %}
```

Comment:
Only adding the `hide_on_click=True` to this panel and not the others because I believe Thibaud intended the dropdown to not be hidden when you select an option whenever possible (for accessibility reasons?). In most cases this is okay as the items are likely links which will take you to the relevant view, but in cases where the option is a button and it opens a modal, you want the dropdown to be hidden because it may obstruct the modal.

See https://github.com/wagtail/wagtail/pull/10557#pullrequestreview-1492775391 and 4bd1f7b267